src/windows.rs

```diff
@@ -64,6 +64,62 @@ pub fn apply_mica(hwnd: HWND) {
   }
 }
 
+
+pub fn clear_blur(hwnd: HWND) {
+  if is_win7() {
+    let bb = DWM_BLURBEHIND {
+      dwFlags: DWM_BB_ENABLE,
+      fEnable: false.into(),
+      hRgnBlur: HRGN::default(),
+      fTransitionOnMaximized: 0,
+    };
+    unsafe {
+      let _ = DwmEnableBlurBehindWindow(hwnd, &bb);
+    }
+  } else {
+    unsafe {
+      set_window_composition_attribute(hwnd, AccentState::Disabled);
+    }
+  }
+}
+
+pub fn clear_acrylic(hwnd: HWND) {
+  if is_win7() {
+    let bb = DWM_BLURBEHIND {
+      dwFlags: DWM_BB_ENABLE,
+      fEnable: false.into(),
+      hRgnBlur: HRGN::default(),
+      fTransitionOnMaximized: 0,
+    };
+    unsafe {
+      let _ = DwmEnableBlurBehindWindow(hwnd, &bb);
+    }
+  } else {
+    unsafe {
+      set_window_composition_attribute(hwnd, AccentState::Disabled);
+    }
+  }
+}
```

Comment:
Acrylic is only supported on Win 10+, remove the Windows 7 logic. Also add checks using `is_win11_dwmsbt` and `is_suplorted_win10()`, same as `apply_acrylic`

Reply:
Oh, woops. Apparently just accidentally copied the logic from blur when I spread the functions apart.